allennlp/commands/fine_tune.py

```diff
@@ -57,6 +57,12 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
                                help='a JSON structure used to override the training configuration '
                                '(only affects the config_file, _not_ the model_archive)')
 
+        subparser.add_argument('--extend-vocab',
+                               action='store_true',
+                               default=False,
+                               help='if specified, we will use the instances in your new dataset to '
+                                    'extend your vocabulary. ')
```

Comment:
maybe mention that embedding expansion isn't implemented yet, so this will likely break if your embedding doesn't include the tokens in your fine-tuning data but